backends/p4tools/common/lib/taint.cpp

```diff
@@ -29,10 +29,20 @@ static bitvec computeTaintedBits(const SymbolicMapType &varMap, const IR::Expres
     CHECK_NULL(expr);
     // TODO: Replace these two with IR::StateVariable.
     if (const auto *member = expr->to<IR::Member>()) {
```

Comment:
Nit: Move this into a small helper function?

Reply:
Yes, will try to clean up once I get rid of IR::Member in Testgen